pallets/xyk/src/lib.rs

```diff
@@ -395,11 +396,12 @@ decl_module! {
             first_asset_id: TokenId,
             second_asset_id: TokenId,
             first_asset_amount: Balance,
+            expected_second_asset_amount: Balance,
```

Comment:
Here and everywhere new parameter was added:
Make new parameter optional to avoid breaking the api